docs/topics/storage.md

```diff
@@ -0,0 +1,64 @@
+# How Brigade uses Kubernetes Persistent Storage
+
+Brigade allows script authors to declare two kinds of storage:
+
+- per-job caches, which persist across builds
+- per-build shared storage, which exists as long as the build is running
+
+Usage of these is described within the [javascript.md](JavaScript docs) and the
+[scripting guide](scripting.md).
+
+This document describes the underlying Kubernetes architecture of these two
+storage types.
+
+## Brigade and PersistentVolumeClaims
+
+Brigade provisions storage using Kubernetes PVCs. Both caches and shared storage
+are PVC-backed.
+
+### Caches
+
+For a Cache, the Brigade worker will check to see if a Job asks for a cache. If it
+does, the worker will create a PVC (if it doesn't already exist) and then mount
+it to the cache.
+
+> A Job, in this case, gains its identity from its name, and the project that
```

Comment:
Is there a particular reason this paragraph is quoted via right angle brackets (>)?

Reply:
Isn't that how you do callouts in MD?

Reply:
Yes it is. I just wanted to make sure that this was intentional. If so, carry on!